client/cmd/sync_publication_contributors.go

```diff
@@ -0,0 +1,93 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"time"
+
+	"github.com/spf13/cobra"
+	api "github.com/ugent-library/biblio-backoffice/api/v1"
+	"github.com/ugent-library/biblio-backoffice/client/client"
+	"google.golang.org/grpc/status"
+)
+
+func init() {
+	SyncPublicationContributorsCmd.Flags().BoolP("noop", "n", false, "inspect changes. Do not execute them.")
+	PublicationCmd.AddCommand(SyncPublicationContributorsCmd)
+}
+
+var SyncPublicationContributorsCmd = &cobra.Command{
+	Use:   "sync-publication-contributors",
+	Short: "Synchronize publication contributor attributes with person service",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return SyncPublicationContributors(cmd, args)
+	},
+}
+
+func SyncPublicationContributors(cmd *cobra.Command, args []string) (err error) {
+
+	// --noop : dry mode
+	noop, _ := cmd.Flags().GetBool("noop")
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
```

Comment:
Use the new `client.Transmit()` function here, instead of the old way  `client.Create`